src/styles/utils/_variables.scss

```
@@ -0,0 +1,7 @@
+$button-colour : #00acdc ;
```

make sure to fix it all over
```suggestion
$button-colour : #00acdc;
```